Source/ProjectFirma.Web/Controllers/DNRUplandRegionController.cs

```diff
@@ -93,9 +91,10 @@ public ViewResult Detail(DNRUplandRegionPrimaryKey dnrUplandRegionPrimaryKey)
                 .SelectMany(x => x.PerformanceMeasureActuals)
                 .Select(x => x.PerformanceMeasure).Distinct()
                 .OrderBy(x => x.PerformanceMeasureDisplayName)
-                .ToList();
+            .ToList();
 
-            var viewData = new DetailViewData(CurrentPerson, region, mapInitJson, viewGoogleChartViewData, performanceMeasures);
+            var coordinator = HttpRequestStorage.DatabaseEntities.People.SingleOrDefault(x=>x.PersonID == region.DNRUplandRegionCoordinator)?.FullNameFirstLastAndOrgShortName;

```

Comment:
I am guessing you had to do this because the generator either isn't smart enough to realize its an FK or there was a conflict with creating a "DnrUpladnRegionCoordinator" object cause that is what the field was called. Once you rename the field on the table to "DnrUpladnRegionCoordinatorID", the generator should provide you with a "DnrUpladnRegionCoordinator" object that is the person and no need to dig into the DB to find the person

Reply:
it did!